### DISCUSSION TITLE
doesn't show an option to create tables in the editor

### DISCUSSION THREAD

**Discussion by bugsecuser** [avitorio/outstatic#210] [Q&A] [answered]
Hey @avitorio, 

How to add tables in post? Currently it does not show such an option.

Thanks!

**Comment by avitorio** [comment]
Hey @bugsecuser,

Thanks for reaching out. Yes, unfortunately we still don't support tables, which is a shame.
I'm going to be working on it.

This is actually a good first issue! Would you be interested in contributing to the project?

**Reply by avitorio** [reply, marked as answer]
Hey @bugsecuser just wanted to let you know that we now have table support.
Could you download the latest version of Outstatic and test it?
Let me know what you think. There's room for improvement, but I think it's good enough for now.
Cheers!

**Reply by bugsecuser** [reply]
Hi @avitorio  Let me first thank you for the effort you are taking for this. 💯 and it now looks great!

**Reply by bugsecuser** [reply]
@avitorio Sorry for the trouble in my test. The same look as when editing the tables is not available when it comes to the result.

**Reply by avitorio** [reply]
Sorry. I don't get it. Please be more specific.

**Reply by avitorio** [reply]
@bugsecuser I think you meant tables weren't rendering on the frontend. I just pushed a fix. Cheers!
